src/sentry/integrations/slack/notify_action.py

```diff
@@ -128,13 +140,33 @@ def get_channel_id(self, integration_id, channel_name):
 
         session = http.build_session()
         resp = session.get('https://slack.com/api/channels.list', params=payload)
-        resp.raise_for_status()
         resp = resp.json()
         if not resp.get('ok'):
             self.logger.info('rule.slack.channel_list_failed', extra={'error': resp.get('error')})
             return None
 
-        return {c['name']: c['id'] for c in resp['channels']}.get(channel_name)
+        channel_id = {c['name']: c['id'] for c in resp['channels']}.get(name)
+
+        if channel_id:
+            return (CHANNEL_PREFIX, channel_id)
+
+        # Look for user ID
```

Comment:
Looking up user ID second since it's more likely they'll be putting in channels first.